grpc/build.gradle

```diff
@@ -2,18 +2,25 @@ import static org.gradle.api.file.DuplicatesStrategy.*
 
 plugins {
     id 'com.exactpro.th2.artifacts-convensions'
-    id "com.google.protobuf" version "0.8.15"
+    id "com.google.protobuf" version "0.9.2"
 }
 
 ext {
-    grpcVersion         = '1.32.1'
-    protobufVersion     = '3.13.0'
-    genDir              = file("${project.projectDir}/src/gen")
-    serviceGeneratorVersion = '3.2.2'
+    grpcVersion         = '1.48.1'
+    protobufVersion     = '3.21.12'
+    serviceGeneratorVersion = '3.3.0'
+    genBaseDir = file("${buildDir}/generated/source/proto") // from protobuf plugin
+}
+
+group = 'com.exactpro.th2'
+version = release_version
+String suffix = property('version_suffix').toString()
+if (!suffix.isEmpty()) {
+    version = "$version-$suffix"
 }
```

Comment:
Will be applied from `buildSrc/src/main/groovy/com.exactpro.th2.common-convensions.gradle`